robot_pose[] robot_name_pose
